woocommerce/woocommerce
# only hide price filter widget when archive has no products

### All Submissions:


### Changes proposed in this Pull Request:

Update the price filter widget to only hide the widget when the unfiltered archive has no products.

Closes #22063 .

### How to test the changes in this Pull Request:

1. Go to a product category that has no products
2. Price filter should not be visible
3. On a product archive that has products,
4. Change min & max price to be the same
5. Change min & max price to a range that has no products
6. For both 4 & 5 the widget should still be visible and functional.

### Other information:




### Changelog entry

> Always show the price filter widget when filtering products by price.
